api/features_api.py

```diff
@@ -352,7 +352,7 @@ def do_delete(self, **kwargs) -> dict[str, str]:
     """Delete the specified feature."""
     # TODO(jrobbins): implement undelete UI.  For now, use cloud console.
     feature_id = kwargs.get('feature_id', None)
-    feature = self.get_specified_feature(feature_id=feature_id)
+    feature: FeatureEntry = self.get_specified_feature(feature_id=feature_id)
     if feature is None:
       return {'message': 'ID does not match any feature.'}
```

Comment:
Nit: this check is not needed, since get_specified_feature() will abort if the feature is not found.

Reply:
Good point.  Gone.